templates/projects/tasks_browse.webpack.ejs

```diff
@@ -32,6 +32,7 @@
               <li><strong>{{_('Gold Tasks')}}</strong>: {{_('Shows button to make a non-gold task gold or update a current gold task')}}.</li>
               {% endif %}
               <li><strong>{{_('Actions')}}</strong>: {{_('Three current actions: download, view, or get user info for task results')}}.</li>
+              <li><strong>{{_('Locks')}}</strong>: {{_('Shows the user locked into the task')}}.</li>
```

Comment:
Can it be rephrased to
`List users working on the task`
which also makes me think if we should change label from `Locks` to `Active Users` or similar.